src/auto/settings.rs

```diff
@@ -43,13 +44,16 @@ impl Settings {
         }
     }
 
-    //#[cfg_attr(feature = "v3_16", deprecated)]
-    //pub fn install_property(pspec: /*Ignored*/&glib::ParamSpec) {
-    //    unsafe { TODO: call gtk_sys:gtk_settings_install_property() }
-    //}
+    #[cfg_attr(feature = "v3_16", deprecated)]
+    pub fn install_property(pspec: &glib::ParamSpec) {
+        assert_initialized_main_thread!();
+        unsafe {
+            gtk_sys::gtk_settings_install_property(pspec.to_glib_none().0);
```

Comment:
> gtk_settings_install_property has been deprecated since version 3.16 and should not be used in newly-written code.
This function is not useful outside GTK+.

Reply:
So let's not add it :)

Reply:
removed it